### ISSUE TITLE
Audiophonics I-Sabre 9038Q2M DAC driver | Now in kernel

### ISSUE THREAD

**Issue by Fourdee** [MichaIng/DietPi#2690]
https://github.com/raspberrypi/linux/commit/d65a0f76d3adcf86a6f5c614c68edb3aeb3b8590

- Remove compile method.
- request/prompt/do rpi-update during selection of audio card.

**Comment by Fourdee** [comment]
- q2m driver compatible with k2m?

**Comment by Fourdee** [comment]
Pending ```rpi-update```:
```
root@DietPi:~# uname -a
Linux DietPi 4.19.32-v7+ #1210 SMP Tue Apr 2 14:07:04 BST 2019 armv7l GNU/Linux
```

**Comment by MichaIng** [comment]
@Fourdee 
Actually I am not keen to add `rpi-update` again. This just ships a rats tail and it is still beta/unstable/testing. As long as the custom build works, I would leave it as is and remove after RPi kernel 4.19 has been released to stable. This should not take long anymore.

What we could try is to simply copy the dtbo from an rpi-updated system and check whether it's compatible with the current stable 4.14 kernel or not?
__________
The big problem with `rpi-update` is still:
- APT kernel packages need to be set on hold to not let them overwrite/downgrade the rpi-updated kernel.
- Once the firmware master is merged into stable (APT packages), the rpi-updated kernel is obsolete.
- But user will never know that. And as it's most likely that user does not run rpi-update manually regularly (which is anyway not recommended if there is no special reason), then the system will stay on an outdated kernel.
- So yeah, first the kernel is beta/unstable and later the kernel is outdated, both without knowledge of the user.
- **So as a result I think `rpi-update` should only be used manually by end users how know exactly what they are doing, how to re-run/update to current master (or "next") branch and how to revert to stable APT packages.**

**Comment by Fourdee** [comment]
@MichaIng 

> I would leave it as is and remove after RPi kernel 4.19 has been released to stable. 

Agree :+1: 

> What we could try is to simply copy the dtbo from an rpi-updated system and check whether it's compatible with the current stable 4.14 kernel or not?

Not viable, would also need to copy the modules used by the overlay. Too little messy/hacky.

**Comment by MichaIng** [comment]
Firmware update just arrived, sadly still on Kernel 4.14 😢.

`Waiting for user reply` = Waiting for 4.19 kernel package release. In any case we keep this in current milestone to not forget.

**Comment by MichaIng** [comment]
@Fourdee 
Yey, RPi kernel 4.19 has just been released: https://github.com/Hexxeh/rpi-firmware/tree/stable
As I am anyway processing the RPi images I will redo them directly with this kernel.

~But it has not yet been pushed to the APT repo 🤔... okay still requires waiting...~
**EDIT: It has been synced to the Stretch repo of archive.raspberrypi.org but not yet to the Buster repo.**
- A bid similar to Kodi 18. Looks like the Buster branch is not yet really in use. Best solution is to add both branches on Raspbian Buster, so the most current package is pulled regardless of branch.

**EDIT2: Ähm, now the deb files have been removed. In the middle of pulling suddenly the packages are not found, now also in browser not available anymore, but still listed in the `packages` file... What's going on there.... Looks like I have to wait a night until everything is solved...**

**Comment by MichaIng** [comment]
Possibly related: https://github.com/MichaIng/DietPi/issues/2855

Actually the dtoverlay is a different one:
- `i-sabre-q2m` vs `i-sabre-k2m`

So selecting `i-sabre-k2m` from dietpi-config like the initial PR intended it will not work. So either, if those are compatible, we need to only allow `i-sabre-q2m` selection, or internally rewrite it.
________
Jep the built-in overlay is compatible: https://github.com/MichaIng/DietPi/issues/2855#issuecomment-495954764
So we should offer this one only then and add to the menu text that this is for i-sabre-k2m as well.

**Comment by MichaIng** [comment]
Resolved with: https://github.com/MichaIng/DietPi/pull/2920
